QUICKSTART.md

```diff
@@ -421,14 +414,17 @@ a particular backend. OpenTelemetry offers five exporters out of the box:
 Other exporters can be found in the [OpenTelemetry Registry].
 
 ```java
-ManagedChannel jaegerChannel =
-    ManagedChannelBuilder.forAddress([ip:String], [port:int]).usePlaintext().build();
+    ManagedChannel jaegerChannel = ManagedChannelBuilder.forAddress("localhost", 3336)
+      .usePlaintext()
+      .build();
+    
     JaegerGrpcSpanExporter jaegerExporter = JaegerGrpcSpanExporter.builder()
-      .setServiceName("example").setChannel(jaegerChannel).setDeadline(30000)
-      .build()
+      .setChannel(jaegerChannel).setTimeout(30, TimeUnit.SECONDS)
```

Comment:
How about updating this to `setEndpoint("localhost:3336")`?

Reply:
done